src/lib.rs

```diff
@@ -291,6 +299,76 @@ where
         let writer = self.make_writer.make_writer();
         bufs.flush_current_buf(writer)
     }
+
+    fn get_timestamp<S>(&self, id: &Id, ctx: &Context<S>) -> Option<String>
+    where
+        S: Subscriber + for<'span> LookupSpan<'span>,
+    {
+        match ctx.span(id) {
```

Comment:
remove the match, now that you got a function where you can use `?`

```suggestion
        let ctx = ctx.span(id)?;
```

Reply:
Fixed as suggested.